source/src/test/cs_TestCrownstoneCentral.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ * Author: Crownstone Team
+ * Copyright: Crownstone (https://crownstone.rocks)
+ * Date: Apr 14, 2021
+ * License: LGPLv3+, Apache License 2.0, and/or MIT (triple-licensed)
+ */
+
+#include <test/cs_TestCrownstoneCentral.h>
+#include <ble/cs_CrownstoneCentral.h>
+#include <events/cs_EventDispatcher.h>
+#include <util/cs_Utils.h>
+#include <storage/cs_State.h>
+#include <drivers/cs_RNG.h>
+
+TestCrownstoneCentral::TestCrownstoneCentral() {}
+
+void TestCrownstoneCentral::init() {
+	EventDispatcher::getInstance().addListener(this);
```

Comment:
Remove include to dispatcher and instead use this EventListener method:
```suggestion
	this->listen();
```

Reply:
Changed to `listen();`.